samples/app-anonymous-users/csharp/Readme.md

```diff
@@ -21,11 +21,6 @@ This sample shows anonymous users support in meeting apps.
 **Interaction with app**
 ![appanonymoususersGif](Images/anonymoususersupport.gif)
 
-## Try it yourself - experience the App in your Microsoft Teams client
-Please find below demo manifest which is deployed on Microsoft Azure and you can try it yourself by uploading the app package (.zip file link below) to your teams and/or as a personal app. (Sideloading must be enabled for your tenant, [see steps here](https://docs.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading)).
```

Comment:
We can remove .zip manifest if it's not working

Reply:
Now its not working .i have remove this line.